### ISSUE TITLE
More explanation of MAINTAIN_SPREADS is desired.

### ISSUE THREAD

**Issue by metaperl** [BitMEX/sample-market-maker#115]
The `settings.py` file states the following for the variable `MAINTAIN_SPREADS`:

> # If True, market-maker will place orders just inside the existing spread and work the interval % outwards,
> # rather than starting in the middle and killing potentially profitable spreads.

I would appreciate a bit more explanation and perhaps an example for this. My specific question is presuming `XBTUSD Ticker: Buy: 7481.5, Sell: 7482.0` what would happen if market-maker placed orders just inside this existing spread? What would happen if it started in the middle?

**Comment by AlexADittrich** [comment]
In the case of `XBTUSD Ticker: Buy: 7481.5, Sell: 7482.0`  one tick is .5. The marketmaker cannot place a order inside this spread.

Another example. If 'XBTUSD Ticker: Buy: 7485.0, Sell: 7495.0` with 'MAINTAIN_SPREADS' = 'TRUE' , marketmaker will not place an bid ask pair "inside" or between 7485 and 7495.

**Comment by metaperl** [comment]
Thank you for your reply Alex. A few more questions:

> In the case of `XBTUSD Ticker: Buy: 7481.5, Sell: 7482.0` one tick is .5. The marketmaker cannot place a order inside this spread.

Why is that? What prevents that?

> Another example. If `XBTUSD Ticker: Buy: 7485.0, Sell: 7495.0` with `MAINTAIN_SPREADS' = 'TRUE' `, marketmaker will not place an bid ask pair "inside" or between 7485 and 7495.

1. What is "potentially profitable" about the spread between 7485 and 7495? 
1. Would a buy or sell be placed between this spread?

**Comment by AlexADittrich** [comment]
No, it does not have to do with the `MIN_SPREAD` parameter. The .5 USD tick is the tick size as set by Bitmex. 

The spread on the order book is a critical component to market making. This isn't an appropriate place to discuss the concept of market making. I recommend you research further into what is market making (there are plenty of informational articles and videos that focus on traditional markets), what role market makers play in the market, and general market making stratgey. 

2,Would a buy or sell be placed between this spread with `MAINTAIN_SPREADS' = 'FALSE'?` 
A. yes

**Comment by metaperl** [comment]
@ryanfox - Before I work on this codebase to document it and supply patches, I need to understand the current functionality. The documentation as it stands was not adequate for me and likely will not be for others who want to contribute. 

@AlexADittrich  - could you link to some specific material that would make these concepts clear?
